airflow/jobs.py

```diff
@@ -119,8 +119,8 @@ def kill(self, session=None):
         job.end_date = timezone.utcnow()
         try:
             self.on_kill()
-        except:
```

Comment:
What if we used [logging.exception](https://docs.python.org/3/library/logging.html#logging.exception) and let Python take care of formatting the exception and stack trace?

```python
except Exception:
    self.log.exception('on_kill() method failed')
```